src/Nethermind/Nethermind.BeaconNode/BeaconChainUtility.cs

```diff
@@ -8,7 +8,10 @@
 using Nethermind.BeaconNode.Containers;
 using Nethermind.BeaconNode.Services;
 using Nethermind.BeaconNode.Ssz;
+using Nethermind.Core2.Crypto;
 using Nethermind.Core2.Types;
+using Nethermind.Logging;
```

Comment:
I'd really like to discuss before you add your own Logging infrastructure. Can we discuss separately?

Reply:
sure - I did not add the logging infra - just extension methods - we still use Microsoft Logger in Eth2 and I am ok to keep it - I just want to keep extensions methods and Eth1 loggers in Nethermind.Logging